WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsTimezoneDialog.java

```diff
@@ -0,0 +1,338 @@
+                }
+            }
+        };
+
+        StringRequest request = new StringRequest(Constants.URL_TIMEZONE_ENDPOINT, listener, errorListener) {
+            @Override
+            protected Map<String, String> getParams() throws AuthFailureError {
+                return RestClientUtils.getRestLocaleParams(getActivity());
+            }
+        };
+
+        showProgressView(true);
+        RequestQueue queue = Volley.newRequestQueue(getActivity());
+        queue.add(request);
+    }
+
+    private void loadTimezones(@NonNull String responseJson) {
+        ArrayList<Timezone> timezones = new ArrayList<>();
+        try {
+            JSONObject jsonResponse = new JSONObject(responseJson);
+            JSONArray jsonTimezones = jsonResponse.getJSONArray("timezones");
+            for (int i = 0; i < jsonTimezones.length(); i++) {
+                JSONObject json = jsonTimezones.getJSONObject(i);
+                timezones.add(
+                        new Timezone(json.getString("label"), json.getString("value"))
+                );
+            }
+
+            mAdapter = new TimezoneAdapter(timezones);
+            mListView.setAdapter(mAdapter);
+            mSearchView.setEnabled(true);
+
+            new Handler().postDelayed(new Runnable() {
+                @Override
+                public void run() {
+                    int index = mAdapter.indexOfValue(mSelectedTimezone);
+                    if (index > -1) {
+                        mListView.setSelection(index);
+                    }
+                }
+            }, 100);
+
+        } catch (JSONException e) {
+            AppLog.e(AppLog.T.SETTINGS, "Error parsing timezones", e);
+            dismissWithError();
+        }
+    }
+
+    private void dismissWithError() {
+        ToastUtils.showToast(getActivity(), R.string.site_settings_timezones_error);
+        dismiss();
+    }
+
+    @Override
+    public void onClick(DialogInterface dialog, int which) {
+        mConfirmed = which == DialogInterface.BUTTON_POSITIVE;
+        dismiss();
+    }
+
+    @Override
+    public void onDismiss(DialogInterface dialog) {
+        Fragment target = getTargetFragment();
+        if (mConfirmed && target != null && !TextUtils.isEmpty(mSelectedTimezone)) {
+            Intent intent = new Intent().putExtra(KEY_TIMEZONE, mSelectedTimezone);
+            target.onActivityResult(getTargetRequestCode(), Activity.RESULT_OK, intent);
+        }
+
+        super.onDismiss(dialog);
+    }
+
+    private void showEmptyView(boolean show) {
+        if (isAdded()) {
+            mEmptyView.setVisibility(show ? View.VISIBLE : View.GONE);
+        }
+    }
+
+    private void showProgressView(boolean show) {
+        if (isAdded()) {
+            mProgressView.setVisibility(show ? View.VISIBLE : View.GONE);
+        }
+    }
+
+    private void hideSearchKeyboard() {
+        if (isAdded()) {
+            ActivityUtils.hideKeyboardForced(mSearchView);
```

Comment:
This is one of those sub-library issues, but I am getting an error for the `hideKeyboardForced` because AS index thinks it doesn't exist, whereas it actually does.

I can't actually remember how we were resolving such issues, but I think it involved pushing the changes in the main library or something like that.